src/raven.js

```diff
@@ -168,6 +170,7 @@ Raven.prototype = {
             this._isRavenInstalled = true;
         }
 
+        Error.stackTraceLimit = this._globalOptions.stackTraceLimit;
```

Comment:
Do we really need to modify this? Why is it not sufficient to just `slice` frames before transmitting?

Note this means that multiple error instances of Raven.js will clobber each other's `Error.stackTraceLimit`.